sormas-app/app/src/main/java/de/symeda/sormas/app/contact/edit/ContactEditFragment.java

```diff
@@ -116,6 +116,13 @@ private void setUpFieldVisibilities(FragmentContactEditLayoutBinding contentBind
 			contentBinding.createCase.setVisibility(GONE);
 		}
 
+		if (record.isMultiDayContact()) {
```

Comment:
showing/hiding firstContact date can be solved in a more simple way using `app:dependencyParentField` and `app:dependencyParentValue` properties in the layout xml